packages/tao/src/commands/migrate.ts

```diff
@@ -550,6 +554,21 @@ async function generateMigrationsJsonAndUpdatePackageJson(
 class MigrationEngineHost extends NodeModulesEngineHost {
   constructor() {
     super();
+
+    this.registerTaskExecutor(
+      {
+        name: NodePackageName,
+        create: options =>
+          import('@angular-devkit/schematics/tasks/node-package/executor').then(
+            mod => mod.default(options)
+          )
+      },
+      {
+        allowPackageManagerOverride: true
+        // packageManager: options.packageManager,
```

Comment:
@vsavkin Does `nx migrate` support passing the packageManager ? Couldn't find any code handling it or passing it to the `BaseWorkflow`